com.github.bilelmoussaoui.Authenticator.json

```diff
@@ -0,0 +1,122 @@
+ {
+   "app-id": "com.github.bilelmoussaoui.Authenticator",
+   "runtime": "org.gnome.Platform",
+   "runtime-version": "3.26",
+   "sdk": "org.gnome.Sdk",
+   "command": "authenticator",
+   "finish-args": [
+     /* X11 + XShm */
+     "--share=ipc", "--socket=x11",
+     /* Wayland */
+     "--socket=wayland",
+     /* Filesystem */
+     "--filesystem=~/.config/Authenticator:ro",
+     /* Keyring */
+     "--talk-name=org.freedesktop.secrets",
+     /* dconf */
+     "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+     "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+   ],
+   "build-options": {
+     "cflags": "-O2 -g",
+     "cxxflags": "-O2 -g",
+     "env": {
+       "V": "1"
+     }
+   },
+   "modules": [{
+       "name": "zbar",
```

Comment:
this needs a reconfigure to fix aarch64

Reply:
Have added `"rm-configure": true`. Is that enough?

Reply:
It doesn't really work with reconfigure on x86_64
```
libtoolize: putting auxiliary files in AC_CONFIG_AUX_DIR, 'config'.
libtoolize: linking file 'config/ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'config'.
libtoolize: linking file 'config/libtool.m4'
libtoolize: linking file 'config/ltoptions.m4'
libtoolize: linking file 'config/ltsugar.m4'
libtoolize: linking file 'config/ltversion.m4'
libtoolize: linking file 'config/lt~obsolete.m4'
configure.ac:10: installing 'config/compile'
configure.ac:6: installing 'config/missing'
automake: warnings are treated as errors
gtk/Makefile.am.inc:13: warning: '%'-style pattern rules are a GNU make extension
Makefile.am:32:   'gtk/Makefile.am.inc' included from here
gtk/Makefile.am.inc:17: warning: '%'-style pattern rules are a GNU make extension
Makefile.am:32:   'gtk/Makefile.am.inc' included from here
pygtk/Makefile.am.inc:20: warning: '%'-style pattern rules are a GNU make extension
Makefile.am:35:   'pygtk/Makefile.am.inc' included from here
qt/Makefile.am.inc:11: warning: '%'-style pattern rules are a GNU make extension
Makefile.am:39:   'qt/Makefile.am.inc' included from here
qt/Makefile.am.inc:14: warning: '%'-style pattern rules are a GNU make extension
Makefile.am:39:   'qt/Makefile.am.inc' included from here
test/Makefile.am.inc:60: warning: '%'-style pattern rules are a GNU make extension
Makefile.am:45:   'test/Makefile.am.inc' included from here
Makefile.am:70: warning: '%'-style pattern rules are a GNU make extension
Makefile.am:73: warning: '%'-style pattern rules are a GNU make extension
/usr/share/automake-1.15/am/ltlibrary.am: warning: 'gtk/libzbargtk.la': linking libtool libraries using a non-POSIX
/usr/share/automake-1.15/am/ltlibrary.am: archiver requires 'AM_PROG_AR' in 'configure.ac'
Makefile.am:32:   'gtk/Makefile.am.inc' included from here
gtk/Makefile.am.inc:1:   while processing Libtool library 'gtk/libzbargtk.la'
/usr/share/automake-1.15/am/ltlibrary.am: warning: 'plugin/libzbarplugin.la': linking libtool libraries using a non-POSIX
/usr/share/automake-1.15/am/ltlibrary.am: archiver requires 'AM_PROG_AR' in 'configure.ac'
Makefile.am:43:   'plugin/Makefile.am.inc' included from here
plugin/Makefile.am.inc:1:   while processing Libtool library 'plugin/libzbarplugin.la'
/usr/share/automake-1.15/am/ltlibrary.am: warning: 'pygtk/zbarpygtk.la': linking libtool libraries using a non-POSIX
/usr/share/automake-1.15/am/ltlibrary.am: archiver requires 'AM_PROG_AR' in 'configure.ac'
Makefile.am:35:   'pygtk/Makefile.am.inc' included from here
pygtk/Makefile.am.inc:1:   while processing Libtool library 'pygtk/zbarpygtk.la'
/usr/share/automake-1.15/am/ltlibrary.am: warning: 'python/zbar.la': linking libtool libraries using a non-POSIX
/usr/share/automake-1.15/am/ltlibrary.am: archiver requires 'AM_PROG_AR' in 'configure.ac'
Makefile.am:29:   'python/Makefile.am.inc' included from here
python/Makefile.am.inc:1:   while processing Libtool library 'python/zbar.la'
/usr/share/automake-1.15/am/ltlibrary.am: warning: 'qt/libzbarqt.la': linking libtool libraries using a non-POSIX
/usr/share/automake-1.15/am/ltlibrary.am: archiver requires 'AM_PROG_AR' in 'configure.ac'
Makefile.am:39:   'qt/Makefile.am.inc' included from here
qt/Makefile.am.inc:1:   while processing Libtool library 'qt/libzbarqt.la'
/usr/share/automake-1.15/am/ltlibrary.am: warning: 'zbar/libzbar.la': linking libtool libraries using a non-POSIX
/usr/share/automake-1.15/am/ltlibrary.am: archiver requires 'AM_PROG_AR' in 'configure.ac'
Makefile.am:8:   while processing Libtool library 'zbar/libzbar.la'
Makefile.am: installing 'config/depcomp'
autoreconf: Entering directory `.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force -I config
autoreconf: configure.ac: tracing
autoreconf: running: libtoolize --copy --force
libtoolize: putting auxiliary files in AC_CONFIG_AUX_DIR, 'config'.
libtoolize: copying file 'config/ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'config'.
libtoolize: copying file 'config/libtool.m4'
libtoolize: copying file 'config/ltoptions.m4'
libtoolize: copying file 'config/ltsugar.m4'
libtoolize: copying file 'config/ltversion.m4'
libtoolize: copying file 'config/lt~obsolete.m4'
autoreconf: running: /usr/bin/autoconf --force
autoreconf: running: /usr/bin/autoheader --force
autoreconf: running: automake --add-missing --copy --force-missing
configure.ac:10: installing 'config/compile'
configure.ac:6: installing 'config/missing'
automake: warnings are treated as errors
gtk/Makefile.am.inc:13: warning: '%'-style pattern rules are a GNU make extension
Makefile.am:32:   'gtk/Makefile.am.inc' included from here
gtk/Makefile.am.inc:17: warning: '%'-style pattern rules are a GNU make extension
Makefile.am:32:   'gtk/Makefile.am.inc' included from here
pygtk/Makefile.am.inc:20: warning: '%'-style pattern rules are a GNU make extension
Makefile.am:35:   'pygtk/Makefile.am.inc' included from here
qt/Makefile.am.inc:11: warning: '%'-style pattern rules are a GNU make extension
Makefile.am:39:   'qt/Makefile.am.inc' included from here
qt/Makefile.am.inc:14: warning: '%'-style pattern rules are a GNU make extension
Makefile.am:39:   'qt/Makefile.am.inc' included from here
test/Makefile.am.inc:60: warning: '%'-style pattern rules are a GNU make extension
Makefile.am:45:   'test/Makefile.am.inc' included from here
Makefile.am:70: warning: '%'-style pattern rules are a GNU make extension
Makefile.am:73: warning: '%'-style pattern rules are a GNU make extension
/usr/share/automake-1.15/am/ltlibrary.am: warning: 'gtk/libzbargtk.la': linking libtool libraries using a non-POSIX
/usr/share/automake-1.15/am/ltlibrary.am: archiver requires 'AM_PROG_AR' in 'configure.ac'
Makefile.am:32:   'gtk/Makefile.am.inc' included from here
gtk/Makefile.am.inc:1:   while processing Libtool library 'gtk/libzbargtk.la'
/usr/share/automake-1.15/am/ltlibrary.am: warning: 'plugin/libzbarplugin.la': linking libtool libraries using a non-POSIX
/usr/share/automake-1.15/am/ltlibrary.am: archiver requires 'AM_PROG_AR' in 'configure.ac'
Makefile.am:43:   'plugin/Makefile.am.inc' included from here
plugin/Makefile.am.inc:1:   while processing Libtool library 'plugin/libzbarplugin.la'
/usr/share/automake-1.15/am/ltlibrary.am: warning: 'pygtk/zbarpygtk.la': linking libtool libraries using a non-POSIX
/usr/share/automake-1.15/am/ltlibrary.am: archiver requires 'AM_PROG_AR' in 'configure.ac'
Makefile.am:35:   'pygtk/Makefile.am.inc' included from here
pygtk/Makefile.am.inc:1:   while processing Libtool library 'pygtk/zbarpygtk.la'
/usr/share/automake-1.15/am/ltlibrary.am: warning: 'python/zbar.la': linking libtool libraries using a non-POSIX
/usr/share/automake-1.15/am/ltlibrary.am: archiver requires 'AM_PROG_AR' in 'configure.ac'
Makefile.am:29:   'python/Makefile.am.inc' included from here
python/Makefile.am.inc:1:   while processing Libtool library 'python/zbar.la'
/usr/share/automake-1.15/am/ltlibrary.am: warning: 'qt/libzbarqt.la': linking libtool libraries using a non-POSIX
/usr/share/automake-1.15/am/ltlibrary.am: archiver requires 'AM_PROG_AR' in 'configure.ac'
Makefile.am:39:   'qt/Makefile.am.inc' included from here
qt/Makefile.am.inc:1:   while processing Libtool library 'qt/libzbarqt.la'
/usr/share/automake-1.15/am/ltlibrary.am: warning: 'zbar/libzbar.la': linking libtool libraries using a non-POSIX
/usr/share/automake-1.15/am/ltlibrary.am: archiver requires 'AM_PROG_AR' in 'configure.ac'
Makefile.am:8:   while processing Libtool library 'zbar/libzbar.la'
Makefile.am: installing 'config/depcomp'
autoreconf: automake failed with exit status: 1
Error: module zbar: module zbar: Child process exited with code 1
```

Reply:
The pattern is usually that you need an autoconf.sh with autoreconf -vfi as per: https://github.com/flathub/fr.free.Homebank/blob/91c81b69cfe98bfea5593ba6475c3b69b7308d04/fr.free.Homebank.json#L58

Reply:
I have done that on my local machine and it doesn't work at all.. I'm getting the same error message. See the logs in my latest comment

Reply:
Figured out how to fix that.